spec/integ/crypto/verification.spec.ts

```diff
@@ -1160,6 +1172,296 @@ describe.each(Object.entries(CRYPTO_BACKENDS))("verification (%s)", (backend: st
         );
     });
 
+    describe("Secrets are gossiped after verification", () => {
+        // we use a legacy olm session as the existing session,
```

Comment:
```suggestion
        // we use a legacy olm session as the existing session.
```

https://en.wikipedia.org/wiki/Comma_splice